mmcv/ops/csrc/pytorch/bbox_overlaps.cpp

```diff
@@ -23,6 +34,14 @@ void bbox_overlaps(const Tensor bboxes1, const Tensor bboxes2, Tensor ious,
     bbox_overlaps_cuda(bboxes1, bboxes2, ious, mode, aligned, offset);
 #else
     AT_ERROR("bbox_overlaps is not compiled with GPU support");
+#endif
+#ifdef MMCV_WITH_MLU
+  } else if (bboxes1.device().type() == at::kMLU) {
+    CHECK_MLU_INPUT(bboxes1);
```

Comment:
Has CHECK_MLU_INPUT been defined?

Reply:
`CHECK_MLU_INPUT` will be added with https://github.com/open-mmlab/mmcv/pull/1467